app/models/invitation.rb

```diff
@@ -5,6 +5,9 @@ class Invitation
   field :token, type: String
   belongs_to :event
 
+  validates_presence_of :email
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.